test/create_db.sh

```diff
@@ -3,6 +3,37 @@ set -o errexit
 cd $(dirname $0)/..
 source test/db-common.sh
 
+# posix compliant escape sequence
+esc=$'\033'"["
+
+function print_heading() {
+  echo
+  echo -e "${esc}0;34;1m"$1"${esc}0m"
```

Comment:
```suggestion
  echo -e "${esc}0;34;1m${1}${esc}0m"
```

It's not clear to me why you ended the string to interpolate `$1`, but this should work, I think.